database/migrations/1743075304168_add_organization_status.ts

```diff
@@ -0,0 +1,33 @@
+import { BaseSchema } from "@adonisjs/lucid/schema";
+
+export default class extends BaseSchema {
+  protected tableName = "student_organizations";
+  protected organizationStatus = ["active", "inactive", "dissolved", "unknown"];
+  protected organizationStatusDefault = this.organizationStatus[3];
+
+  async up() {
+    this.schema.alterTable(this.tableName, (table) => {
+      table
+        .enum("organization_status", this.organizationStatus, {
+          useNative: true,
+          enumName: "organization_status",
+          existingType: false,
+        })
+        .defaultTo(this.organizationStatusDefault)
+        .notNullable();
+    });
+    this.defer(async (db) => {
+      await db
+        .from(this.tableName)
+        .update({ organization_status: this.organizationStatusDefault })
+        .whereNull("organization_status");
+    });
```

Comment:
not needed
the `organization_status` column is defined as not nullable with a default value, you'll never find a null value there